week2homework.js

```diff
@@ -0,0 +1,121 @@
+//1
+let EnglishGreeting = "Hello world!";
+console.log (EnglishGreeting)
+let ArabicGreeting = "مرحباً بالعالم";
+console.log (ArabicGreeting)
+let ItalianGreeting = "Ciao Mondo!";
+console.log (ItalianGreeting)
+let TurkishGreeting = "Selam Dünya!";
+console.log (TurkishGreeting)
+let DutchGreeting = "Hallo Wereld!";
+console.log (DutchGreeting)
+
+//2
+console.log ('I\'m awesome');
+
+//3
+let x ;
+console.log ("the value of my variable x will be:EvenNumber");
```

Comment:
Are you sure that the value of `x` is `EvenNUmber` ?
(I don't think it will be that value)